Dockerfile

```diff
@@ -39,15 +50,18 @@ ENV TEST=false
 # Finally used for all misk task, will not be used on prod runtime
 
 FROM base AS tools
+SHELL ["/bin/bash", "-o", "pipefail", "-ceux"]
```

Comment:
What does this ? Maybe add a comment.

Reply:
=> comment added:
# Fail on error on pipe, see: https://github.com/hadolint/hadolint/wiki/DL4006.
# Treat unset variables as an error when substituting.
# Print commands and their arguments as they are executed.